### PR TITLE
Added default build args 

### PR DESCRIPTION
**Pull request by eyduh**
Added default build args which one would update with new releases etc and added USER & GROUP args which can be specified at build time for easier modification to the owner for music folder.

I set the default UID/GID to the standard for most linux distros first non-root user, but this can ofc be overridden at build time.

( I'm considering whether it would be better to use `lscr.io/linuxserver/alpine-base-image` instead of the standard alpine image as the user and group can be set at runtime with PUID/GUID env variables but that is outside the scope for this pr )

### CODE REVIEW

**Comment by ogarcia** [comment]
The `ALPINE_VERSION` and `POLARIS_VERSION` variables are defined in the Makefile so not defining them in the Dockerfile is intentional.

As for changing the default user UID and GID, if you do it the container will not boot without mounting external volumes.

On the other hand, this only affects the moment of the creation of the container, not its execution, so if you are fine with the user being 1000, but I am better off with 1001, I will have to rebuild the image.

In short, if you necessarily have to rebuild the image to modify the UID and GID, then you can directly modify the `Dockerfile` to your liking and do it so this merge request does not have too much scope.
